pkg/crc/config/settings.go

```diff
@@ -46,16 +46,16 @@ func RegisterSettings(cfg *Config) {
 		return ValidateBool(value)
 	}
 
-	validateCPUs := func(value interface{}) (bool, string) {
-		return ValidateCPUs(value, GetPreset(cfg))
+	validCPUs := func(value interface{}) (bool, string) {
```

Comment:
should be `validateCPUs` ?

Reply:
this can't be `validateCPUs` as the name is already used by the function in validations.go. It's the same for `validMemory`/.. The `validCPUs` name is not great, open to suggestions! Maybe rename `validateCPUs` to `validateCPUsForPreset` ?

Reply:
eh, no need, `validCPUs` is also okay :)